main.cpp

```diff
@@ -19,13 +19,20 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
+    CastCallBack(Rewriter& _rewriter):rewriter(_rewriter) {
         // Your code goes here
     };
 
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+        auto csce = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");

```

Comment:
What if `csce` is a null pointer? How to protect the program from crashing in such cases?

Reply:
we can add  condition that compares csce and nullptr